roles/confluent.connect-distributed/defaults/main.yml

```diff
@@ -6,6 +6,8 @@ kafka:
       systemd_override: /etc/systemd/system/confluent-kafka-connect.service.d
       user: cp-kafka-connect
       group: confluent
+      keytab: /etc/security/keytabs
+      realm: confluent.example.com
```

Comment:
this variable doesn't get used anywhere, realm is pulled from the hosts file

Reply:
fixed.